autopush/endpoint.py

```diff
@@ -209,13 +209,16 @@ def _write_unauthorized_response(self, message="Invalid authentication",
         self._write_response(401, errno=109, message=message, headers=headers,
                              **kwargs)
 
-    def _response_err(self, fail):
+    def _response_err(self, fail, caller=None):
         """errBack for all exceptions that should be logged
 
         This traps all exceptions to prevent any further callbacks from
         running.
 
         """
+        if not caller:
+            caller = self.__module__
```

Comment:
Actually, don't we want the class name since this is a base class and we want to know the actual class?